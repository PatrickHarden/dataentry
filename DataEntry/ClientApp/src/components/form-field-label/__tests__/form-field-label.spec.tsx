import React from 'react';
import { shallow } from 'enzyme';

import FormFieldLabel from '../form-field-label';

describe('components', () => {

  describe('form-field-label', () => {

    let props:any;

    beforeEach(() => {
        props = {
            label: 'Test'
        }
    });

    // snapshot render
    it('renders according to snapshot', () => {
        expect(shallow(<FormFieldLabel {...props} />)).toMatchSnapshot();
    });

  });
});