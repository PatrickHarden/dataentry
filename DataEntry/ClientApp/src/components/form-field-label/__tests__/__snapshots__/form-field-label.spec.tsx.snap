// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components form-field-label renders according to snapshot 1`] = `
<styled.h5
  label="Test"
>
  Test
</styled.h5>
`;
